src/styles/_catalog.scss

```
@@ -14,8 +12,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms;
```

specify which properties you are animating

```suggestion
    transition: transform 300ms, color 300ms ;
```